### PR TITLE
Migrate Wallet table to one with BigAutoField

### PR DESCRIPTION
**Pull request by fmrsabino**
- Fix an issue where editing a `Wallet` would create a new one – the reason for this was because `name` was the primary key for `Wallet`. A primary key is ready-only so attempt to change would result in the creation of a new object.
- Change `Wallet.name` to `Wallet.key`
- Introducing an auto-generated ID to an already existing table can tricky as it expects default values for non-null columns (eg.: the new auto generated id). In order to tackle this the following migrations is performed:
  1. Create a new schema `WalletNew` with the columns `id`, `key` (old `name`) and `chains` (Many-to-Many relationship)
  2. Copy data from old schema `Wallet` and insert into the newly created schema `WalletNew`
  3. Clear relationship with between `Wallet <> Chains`
  4. Delete `Wallet` schema
  5. Rename `WalletNew` to `Wallet`

### CODE REVIEW

**Review by Uxio0 — APPROVED** [review]
I think this will make our lifes easier 😉

File: src/chains/migrations/0033_walletnew.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "key",
```
Comment (Uxio0) [inline src/chains/migrations/0033_walletnew.py:41]: Why renaming to `key`? I found `name` more self explaining

Comment (fmrsabino) [reply src/chains/migrations/0033_walletnew.py:36]: This was a suggestion from Ivan. The reason is that `name` might look like a UI specific field whereas `key` is more of an identifier. We can potentially add name to have a string representation of the `Wallet` in the future. What do you think?

**Comment by fmrsabino** [comment]
@Uxio0 I added a test and fixed an issue with the migration (where it was using `name` to set the new fields instead of `key`) 26dce1b

File: src/chains/migrations/0033_walletnew.py (hunk 1 above)
Comment (Uxio0) [reply src/chains/migrations/0033_walletnew.py:36]: No strong opinion, but for me every wallet has a name, nothing related to the UI. Whatever you think is more clear 😃

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**